chirp/chirp_common.py

```diff
@@ -395,7 +395,7 @@ def format_freq(self):
     def parse_freq(self, freqstr):
         """Set the frequency from a string"""
         self.freq = parse_freq(freqstr)
-        return self.freqx
+        return self.freq
```

Comment:
Yikes! Not sure this ever got used, but better that it's right. Looks like an errant typo on my part :(